ganalytics.cpp

```diff
@@ -584,7 +619,7 @@ QNetworkAccessManager *GAnalytics::networkAccessManager() const
  * @param appVersion
  * @param screenName
  */
-void GAnalytics::sendAppView(const QString &screenName)
+void GAnalytics::sendAppView(const QString &screenName, const QMap<QString, QString> &customValues)
```

Comment:
Please use QVariantMap for the customValues parameter since this function may be called from QML, which is unable to construct a QMap<QString, QString>.